### ISSUE TITLE
[Due for payment 2026-01-30] Screen Reader: Settings - Preferences: Non-English options announced incorrectly or not announced

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#74881]
## Action Performed
Prerequisite:
- User is logged in

1. Using JAWS + Chrome, open the URL: https://new.expensify.com/
2. Tab to focus on the "My Settings" button and press Enter to activate it
3. Once the My Settings menu is displayed, tab to focus on the "Preferences" menu item and press Enter to activate it
4. Tab to focus on the "Language" button and press Enter to expand it
5. Use the tab key to focus any of the language options.
6. Observe the screen reader announcement

## Expected Result
Each language option should be announced in its respective language or at least clearly communicated by the screen reader.
The assistive technology should correctly detect the language of the text and announce it appropriately, ensuring users can easily identify and select the intended option.

## Actual Result
When navigating to non-English language options (such as Deutsch, Español, Français, Nederlands, Polski, and Português (BR)), they are announced using an English accent. For some languages (such as 中文 (简体) and 日本語), the screen reader does not announce them at all, skipping the label entirely. This makes it difficult for users relying on assistive technologies—especially screen reader users who are native speakers of other languages—to accurately perceive and select their preferred language option.

HTML:

<div aria-label="中文 (简体)" role="presentation" data-tag="pressable" data-selection-scrapper-hidden-element="true" data-inner-box-shadow-element="false" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21" id="zh-hans" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx" style="background-color: rgb(10, 46, 37);"><div class="css-g5y9jx r-13awgt0 r-1wtj0ep r-1awozwy r-18u37iz r-b5h31w r-1ah4tor r-9ffhgg r-lrvibr r-adacv r-kzbkwu r-ttdzmv" data-testid="base-list-item-zh-hans" style="background-color: rgb(10, 46, 37);"><div class="css-g5y9jx r-13awgt0 r-1habvwh"><div dir="auto" class="css-146c3p1 r-dnmrzs r-1udbk01 r-1iln25a" style="color: rgb(231, 236, 233); font-size: 15px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 700; min-height: 20px; line-height: 20px; white-space: pre; text-decoration-line: none; overflow: hidden;">中文 (简体)</div></div></div></div>

## Area issue was found
Settings-Preferences

## Failed Checkpoint
WCAG 3.1.1 Language of Page
https://www.w3.org/WAI/WCAG22/Understanding/language-of-page

## User Impact
People who use screen readers or other technologies that convert text into synthetic speech benefit when the page language is properly set as assistive technology uses the programmatic language to determine correct pronunciation rules. This also benefits people who find it difficult to read written material with fluency and accuracy, such as recognizing characters and alphabets or decoding words, people with certain cognitive, language and learning disabilities who use text-to-speech software, and people who rely on captions for synchronized media.

## Suggested Resolution
Use the lang attribute on each language option to make sure the component or text element is coded with the correct language so that screen readers can automatically use the right voice and pronunciation.
https://www.w3.org/WAI/WCAG22/Techniques/html/H58
An alternative might be to show each language option together with an option translated into English.

## Environment(s)
Windows 11 Home - Chrome

## Screenshots/Videos
<img width="1918" height="967" alt="Image" src="https://github.com/user-attachments/assets/1f2416dd-2c5a-46a6-8acd-040d1d413575" />

https://github.com/user-attachments/assets/63c47d70-7192-40dc-8e5d-579b63161483

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6995940
**Issue reported by:** NA
**Slack conversation:** NA

<details><summary>Issue Owner</summary>Current Issue Owner: @mkhutornyi</details>

**Comment by mallenexpensify** [comment]
`Accessibility` issue, will be addressed soon.
